packages/serverless/src/gcpfunction/cloud_events.ts

```diff
@@ -72,14 +72,14 @@ function _wrapCloudEventFunction(
       return (fn as CloudEventFunctionWithCallback)(context, newCallback);
     }
 
-    void Promise.resolve()
+    return Promise.resolve()
```

Comment:
Has this fix been tested on GCP using the example that @killthekitten provided in the example?

My only concern with this is that the top level function i.e. L37 is returning a non-asynchronous function to the GCP runner, so the promises within the function might not be correctly resolved? (I might be wrong)

If my above concern is valid, it would be helpful to have the following expectations in the tests:
```
expect(wrappedHandler).toBeInstanceOf(Promise);
```

Reply:
Yes, we tested it and it works.